src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -18,5 +18,6 @@ public enum DocumentFlags
         Reserved3 = 0x80,
 
         HasAttachments = 0x100,
+        Revision = 0x200
```

Comment:
This is actually used to pass around information about the revision, and it isn't actually being stored. We should probably have a better name for this, that indicate that it isn't actually stored.